anchore_engine/common/helpers.py

```diff
@@ -1,12 +1,246 @@
 """
 Common message types and marshalling helper functions
 """
-import base64
 import copy
-import json
 import time
 
+import base64
+import json
+import stat
+
+import anchore_engine.configuration.localconfig
+from anchore_engine import utils
 from anchore_engine.subsys import logger
+from anchore_engine.common import os_package_types
+
+
+def make_image_content_response(content_type, content_data):
```

Comment:
All the methods in this file are copied over from the apiext helpers, to be more explicit about them being used in potentially multiple services

Reply:
They could use a refactor as well but didn't want to do too much in this PR